recipe/meta.yaml

```diff
@@ -144,13 +146,18 @@ outputs:
         - iminuit >=2
         - lmfit
         - lsstdesc.coord
+        - lsst-ts-xml
         - matplotlib-base !=3.6.0
+        - mkl  # [linux64]
         - moto >=3
         - mpi4py
+        - nest_asyncio
         - networkx
+        - ngmix-core
         - numexpr =2
         - {{ pin_compatible('numpy') }}
-        - pandas
+        - panda-client !=1.5.65
+-       - pandas
```

Comment:
Is this the lint?  Hard to see in the diff...

Reply:
Yes, I missed it.  Fixed now.